src/ResourceManager/KeyVault/Commands.KeyVault/Commands/BackupAzureKeyVaultKey.cs

```diff
@@ -27,6 +27,8 @@ namespace Microsoft.Azure.Commands.KeyVault
     [OutputType(typeof(String))]
```

Comment:
Since you are updating the cmdlet, please add ShouldProcess support, if it is not already there